.github/workflows/hawkscan-integration.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: stackhawk/hawkscan-action@main
         with:
           apiKey: ${{ secrets.HAWK_API_KEY }}
-          workspace: ${{ env.GITHUB_WORKSPACE }}/__tests__/integration-tests/configs/${{ github.event.client_payload.repoName }}/
+          workspace: $GITHUB_WORKSPACE/__tests__/integration-tests/configs/${{ github.event.client_payload.repoName }}/
```

Comment:
I think it was ok the way it was before right?  I thought it only whined about it in the action.yml right?

Reply:
Looks like ${{ env.GITHUB_WORKSPACE }} didn't work, so changing to work with $GITHUB_WORKSPACE